services/src/main/java/org/keycloak/broker/oidc/OIDCIdentityProvider.java

```diff
@@ -530,7 +531,7 @@ protected BrokeredIdentityContext extractIdentity(AccessTokenResponse tokenRespo
 
         identity.setEmail(email);
 
-        identity.setBrokerUserId(getConfig().getAlias() + "." + id);
+        identity.setBrokerUserId(issuer + "." + id);
```

Comment:
When you change this, you'll need to migrate existing user sessions, at least the persistent user sessions.